etc/lower-minheartbeatfrequencyms.diff

```diff
@@ -1,5 +1,5 @@
 diff --git a/Sources/CLibMongoC/include/CLibMongoC_mongoc-client-pool.h b/Sources/CLibMongoC/include/CLibMongoC_mongoc-client-pool.h
-index 16d7eb1..bcf1cde 100644
+index 16d7eb1a..bcf1cdee 100644
```

Comment:
this patch needed to be regenerated due to changes in `mongoc-client-pool.{h, c}`